packages/ui/src/views/repo/pull-request/details/components/changes/pull-request-changes-filter.tsx

```diff
@@ -38,9 +45,14 @@ export interface FilterViewProps {
   diffMode: DiffModeEnum
   setDiffMode: (value: DiffModeEnum) => void
   useTranslationStore: () => TranslationStore
+  pullReqCommits?: TypesCommit[]
+  defaultCommitFilter: CommitFilterItemProps
+  selectedCommits: CommitFilterItemProps[]
+  setSelectedCommits: React.Dispatch<React.SetStateAction<CommitFilterItemProps[]>>
+  viewedFiles: number
+  totalFiles: number
 }
 
-// TODO: workon on filter and files viewed
 export const PullRequestChangesFilter: React.FC<FilterViewProps> = ({
```

Comment:
nit: As suggested, component and it's props need to have naming convention in a similar manner. `PullRequestChangesFilterProps`